convex_hull.cpp

```diff
@@ -162,9 +162,12 @@ int main(int argc, char* argv[]) {
 	FILE *ifp, *ofp;
 
 	if(argc < 2){
-		fprintf(stderr, "Error: We Expect atleast one argument, as Input text file.\nUsage - You can give Input text file as argument, you can also give output text in which you want to save output, if you don't provide any output file then result is shown in STDOUT.\n");
+		fprintf(stderr, "Error: We Expect atleast one argument, as Input text file.\n"
+			"Usage - You provide Input text file as argument, you can also give output file"
```

Comment:
Add ` `(space) at the end of this string or at the beginning of the string in the next line.